apitools/org.eclipse.pde.api.tools.ui/src/org/eclipse/pde/api/tools/ui/internal/refactoring/FilterChange.java

```diff
@@ -96,26 +96,6 @@ protected String getDeleteName() {
 		return NLS.bind(RefactoringMessages.FilterChange_remove_used_filter, problem.getMessage());
 	}
 
-	/**
-	 * Returns the name to use for a {@link #RENAME} change operation
-	 *
-	 * @return the name for a {@link #RENAME} change
-	 */
-	protected String getRenameName() {
-		IApiProblem problem = this.filter.getUnderlyingProblem();
-		return NLS.bind(RefactoringMessages.FilterChange_remove_used_filter, new Object[] { problem.getMessage() });
-	}
-
-	/**
-	 * Returns the name to use for a {@link #MOVE} change operation
-	 *
-	 * @return the name for a {@link #MOVE} change
-	 */
-	protected String getMoveName() {
-		IApiProblem problem = this.filter.getUnderlyingProblem();
-		return NLS.bind(RefactoringMessages.FilterChange_remove_used_filter, new Object[] { problem.getMessage() });
-	}
-
```

Comment:
These methods return the very same result as `getDeleteName()` and are never called.